src/apps/dictionaries/redux/actions.ts

```diff
@@ -357,20 +355,27 @@ export const editDictionaryVersionAction = createActionThunk(
 );
 
 export const recursivelyAddConceptsToDictionaryAction = (
-  sourceUrl: string,
   dictionaryUrl: string,
   rawConcepts: (APIConcept | string)[],
-  bulk: boolean = false
+  bulk: boolean = false,
+  sourceUrl?: string
 ) => {
-  return async (dispatch: Function, getState: Function) => {
-    const concepts = rawConcepts.map(concept =>
-      typeof concept === "string"
-        ? {
-            id: concept,
-            url: `${sourceUrl}concepts/${concept}/`
-          }
-        : concept
-    );
+  return async (dispatch: Function, getState: function) => {
```

Comment:
This is the small typo it's complaining about:

```suggestion
  return async (dispatch: Function, getState: Function) => {
```

`function` is a keyword `Function` is a type, if that makes any sense an here (after the `:`) this is serving as a type annotation.